public/app/plugins/datasource/prometheus/querybuilder/components/MetricEncyclopediaModal.tsx

```diff
@@ -410,6 +410,49 @@ export const MetricEncyclopediaModal = (props: Props) => {
     return results ?? 10;
   };
 
+  const ButtonCell = ({
+    row: {
+      original: { value },
+    },
+  }: CellProps<MetricData, void>) => {
+    return (
+      <Button
+        size="sm"
+        aria-label="use this metric button"
+        data-testid={testIds.useMetric}
+        onClick={() => {
+          onChange({ ...query, metric: value });
+          reportInteraction('grafana_prom_metric_encycopedia_tracking', {
+            metric: value,
+            hasVariables: variables.length > 0,
+            hasMetadata: hasMetadata,
+            totalMetricCount: metrics.length,
+            fuzzySearchQuery: fuzzySearchQuery,
+            fullMetaSearch: fullMetaSearch,
+            selectedTypes: selectedTypes,
+            letterSearch: letterSearch,
+          });
+          onClose();
+        }}
+      >
+        Use this metric
+      </Button>
```

Comment:
for this i think having a column of solid blue buttons is too harsh. i would personally prefer one of the following:
```
<Button variant=“primary” fill=“outline” ...rest />
<Button variant=“secondary” fill=“solid” ...rest />
```

feel free to check with UX if you are not sure.

Reply:
Changed to secondary!